src/fileActions.jsx

```diff
@@ -273,11 +258,11 @@
     const [errorMessage, setErrorMessage] = useState(undefined);
 
     let title;
-    if (selected.type === "file") {
+    if (selected.type === "reg") {
         title = cockpit.format(_("Rename file $0"), selected.name);
-    } else if (selected.type === "link") {
+    } else if (selected.type === "lnk") {
         title = cockpit.format(_("Rename link $0"), selected.name);
-    } else if (selected.type === "directory" || selected.items_cnt) {
+    } else if (selected.type === "dir" || selected.items_cnt) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-210-20240126-155112-93335757-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)